templates/internet-of-things/edgex.html

```diff
@@ -0,0 +1,346 @@
+        <p class="p-heading--four" id="subscribe">Subscribe to our weekly newsletter</p>
+        <form action="https://pages.canonical.com/index.php/leadCapture/save" method="post" id="mktoForm_3415" class="p-form--stacked">
+          <div class="p-form__group">
+            <label for="Email">Email:*</label>
+            <input class="u-no-margin--bottom" id="Email" placeholder="Email" name="Email" maxlength="255" type="email" required="">
+          </div>
+          <div class="p-form__group">
+            <label for="Company">Company:</label>
+            <input class="u-no-margin--bottom" id="Company" placeholder="Company (optional)" name="Company" maxlength="255" type="text">
+          </div>
+
+          <p>
+            <small>In submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/newsletter">Canonical’s Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</small>
+          </p>
+          <div class="u-sv3">
+            <div class="g-recaptcha" data-sitekey="{{ CAPTCHA_TESTING_API_KEY }}"></div>
+          </div>
+
+          <p class="u-no-margin--bottom">
+            <input type="hidden" name="Consent_to_Processing__c" value="Yes">
+            <button class="p-button" type="submit">Subscribe now</button>
+          </p>
+
+          <input type="hidden" name="formid" value="3690">
+          <input type="hidden" name="edgeXblogsubscription" value="yes">
+          <input type="hidden" name="lpId" value="">
+          <input type="hidden" name="munchkinId" value="066-EOV-335">
+          <input type="hidden" name="lpurl" value="">
+          <input type="hidden" name="ret" value="https://ubuntu.com/internet-of-things/edgex#subscribed">
+          <input type="hidden" name="return_url" value="https://ubuntu.com/internet-of-things/edgex#subscribed">
+          <input type="hidden" name="cr" value="">
+          <input type="hidden" name="kw" value="">
+          <input type="hidden" name="q" value="">
+        </form>
+      </div>
+
+
+    </div>
+</section>
+
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-12 u-align--center">
+      <h4 class="p-muted-heading u-no-max-width">Proven in the field</h4>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-12">
+      <ul class="p-inline-images u-no-margin--bottom">
+        <li class="p-inline-images__item">
+          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/093c43ef-intel_logo.svg" alt="Intel">
+        </li>
+        <li class="p-inline-images__item">
+          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/c0ee6182-dell_logo.svg" alt="Dell">
+        </li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-12 u-align--center">
+      <a href="#subscribe">Sign up for latest news&nbsp;&rsaquo;</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
```

Comment:
Is there any reason why you don't use `u-fixed-width` instead in these cases?

Reply:
Just personal preference. No difference really.